high-availability.helmfile.yaml

```diff
@@ -50,7 +50,7 @@ releases:
   - name: ratify
     namespace: gatekeeper-system
     chart: ratify/ratify
-    version: 1.15.0-rc.1 # Make sure this matches Chart.yaml
+    version: 1.15.0 # Make sure this matches Chart.yaml
```

Comment:
helm file changes will also need to be updated  on dev , correct?

Reply:
yes, we'll need to update dev branch once the assets published.